test/e2e/network/service.go

```diff
@@ -37,6 +37,7 @@ import (
 	cloudprovider "k8s.io/cloud-provider"
 	"k8s.io/kubernetes/pkg/controller/endpoint"
 	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/framework/endpoints"
```

Comment:
given the other naming patterns we should prefix with e2e to avoid collisions with existing libraries.

Reply:
`e2eendpoints` is a bit weird.. do you have some suggestions?

Reply:
i think it's fine. the double `ee` is not a problem.

Reply:
Thanks Lubomir